src/API/REST/V3/Routes/Donors/DonorController.php

```diff
@@ -332,152 +308,28 @@ public function prepare_item_for_response($item, $request): WP_REST_Response
             ],
         ];
 
-        $response = new WP_REST_Response($item);
+        $response = new WP_REST_Response(Item::formatDatesForResponse($item, ['createdAt']));
         $response->add_links($links);
```

Comment:
We should add links for donations and subscriptions since those are already relationships in the Donor model.

Reply:
Resolved in 51a809640ae4bad9d0e95c50bc97972bf5054c3f and 8de2da07c84dccf2abb629253c7b52ef91e046a3 and c232079fab890f67c1f0627ee96e7d369e2de5a9